src/applications/static-pages/alerts-dismiss-view.js

```diff
@@ -21,43 +21,33 @@ export function alertsBuildShow() {
     content.setAttribute('aria-hidden', ariaHidden);
   }
 
-  const elements = document.querySelectorAll('.usa-alert-text');
-  let i;
-  for (i = 0; i < elements.length; i++) {
-    if (
-      elements[i].id &&
-      document.querySelectorAll(
-        `#${elements[i].id} .field--name-field-text-expander`,
-      ).length
-    ) {
-      const clicker = document.querySelector(
-        `#${elements[i].id} .field--name-field-text-expander`,
-      );
+  Array.from(document.querySelectorAll('.usa-alert-text'))
+    .filter(el => el.querySelector('.field--name-field-text-expander') !== null)
+    .forEach(el => {
+      const clicker = el.querySelector('.field--name-field-text-expander');
 
       // Toggle the expander info
       clicker.addEventListener('click', expanderFire);
-    }
-  }
+    });
 
-  // This is the cookie dismiss logic.
-  const wrapsClicker = document.querySelectorAll('.usa-alert-dismiss');
-  let j;
-  let k;
   let target;
   let frequency;
   let date;
   let expires;
 
   // Build our array of dismissed alerts.
-  const items = document.cookie
+  const dismissedAlerts = document.cookie
     .split(';')
     .filter(c => c.trim().indexOf('usa') === 0)
     .map(c => c.trim().replace(/=dismissed/g, ''));
 
   // Iterate through the alerts array, and dismiss.
-  for (j = 0; j < items.length; j++) {
-    document.getElementById(items[j]).classList.add('dismissed');
-  }
+  dismissedAlerts.forEach(id => {
+    const el = document.getElementById(id);
+    if (el) {
```

Comment:
This is the logic I added. Sometimes these dismissed alerts don't exist on a page.